node/sc/api_bridge.go

```diff
@@ -253,6 +254,21 @@ func (sb *SubBridgeAPI) GetAnchoring() bool {
 	return sb.subBridge.GetAnchoringTx()
 }
 
+func checkBridgeContractVersion(cBridge, pBridge *bridge.Bridge) bool {
+	errStr := "Bridge contract version mistmatched"
+	cv, cvErr := cBridge.VERSION(nil)
```

Comment:
@hyunsooda What is the return value if VERSION is not defined in the contract?

Reply:
Thank you. Good catch. 

I think we need to establish a semi-standard for the contract's version query. As developers would customize bridge contracts in the future, a standard function that returns the bridge contract's version seems needed so that the variable's name can be renamed anything as long as they implement that function. Please add how do you feel.

Reply:
Yeah, I agree with you to some extent. We have [KIP-13](https://kips.klaytn.com/KIPs/kip-13) to query interfaces in a standard way. It does not exactly fit to this, though.

By the way, what I wanted to say is that whether this works well with previously deployed bridge contracts without this function.

Reply:
Bridge registration is failed in this PR If the variable `VERSION` does not exist in the bridge contract.

Reply:
I thought that we don't have VERSION variable for the current bridge contract implementation. It seems that I was wrong, and this version check is good to go. Thanks.